app/controllers/schedules_controller.rb

```diff
@@ -3,6 +3,14 @@ class SchedulesController < ApplicationController
   before_action :respond_to_options
   load_and_authorize_resource :conference, find_by: :short_title
   load_resource :program, through: :conference, singleton: true, except: :index
+  around_action :use_timezone_for_this_request
+
+  # FIXME: The timezome should only be applied on output, otherwise
+  # you get lost in timezone conversions...
+
+  def use_timezone_for_this_request(&block)
```

Comment:
I have just tried to create a new conference with GMT-10:00 time zone and it doesn't work. I scheduled the event like this:
![image](https://cloud.githubusercontent.com/assets/16052290/23695811/3e508898-03e1-11e7-8e2e-2179d263bbb2.png)
and the event time keep being utc in the show page:
![image](https://cloud.githubusercontent.com/assets/16052290/23695844/59577430-03e1-11e7-9c81-ee331ff09287.png)

and the time in the all events is wrong:
![image](https://cloud.githubusercontent.com/assets/16052290/23695802/330c963e-03e1-11e7-87b9-7fe582c57065.png)

It could be that you tried it with a conference which timezone is UTC.